### ISSUE TITLE
Pi-hole Update broke compatibility 

### ISSUE THREAD

**Issue by SythenElexia** [sterrenburg/flutterhole#161]
Recent update to pihole broke compatibility with flutterhole and can no longer interact with it (pausing and viewing logs) but can still load statistics.

API Token and any auth details fail to validate.

 - Pi-hole version: 5.12
 - FTL: 5.17
 - FlutterHole version: 5.2.3

**Comment by jwesemann** [comment]
Same behavior here.

**Comment by Ngiri67** [comment]
Thank goodness I'm not the only one. I have the same here.

**Comment by iceandrews** [comment]
Yeah this is back :(

**Comment by SythenElexia** [comment]
Updated Pihole web interface to 5.14.2 and flutterhole seems to be working again

**Comment by iceandrews** [comment]
Yeah looks to be fixed with https://github.com/pi-hole/AdminLTE/commit/d68d5eee9a9c03c559aabb569c1a2cd18c60dafa
